pkg/engine/handlers/mutation/common.go

```diff
@@ -69,15 +69,22 @@ func (f *forEachMutator) mutateElements(ctx context.Context, foreach kyvernov1.F
 
 	patchedResource := f.resource
 	var allPatches []jsonpatch.JsonPatchOperation
+	reverse := false
 	if foreach.RawPatchStrategicMerge != nil {
+		reverse = true
```

Comment:
Why reverse the order for `PatchStrategicMerge`?

Reply:
Looks like it's needed by kustomize